### ISSUE TITLE
Forms error message aria-described by

### ISSUE THREAD

**Issue by micmath** [bbc/gel#147]
via @JamieKnightBBC ... When the error message is populated in the reference implementation, there should be an `arai-live` role to announce the change rather than relying on the screen reader noticing a change only via `aria-described-by`.
